fix checksum accordions (#16441)
docker/docs: desktop/release-notes.md

#### For all platforms

> - Docker Desktop now requires an internal ne2rk subnet of size /24. If you were previously using a /28, it is automatically expanded to /24. If you experience ne2rking issues, check to see if you have a clash between the Docker subnet and your infrastructure. Fixes [docker/for-win#13025](https://github.com/docker/for-win/issues/13025).

Reviewer: ```suggestion
- Docker Desktop now requires an internal network subnet of size /24. If you were previously using a /28, it is automatically expanded to /24. If you experience networking issues, check to see if you have a clash between the Docker subnet and your infrastructure. Fixes [docker/for-win#13025](https://github.com/docker/for-win/issues/13025).
```

- Fixed issues related to performing actions on selected containers. [Fixes https://github.com/docker/for-win/issues/13005](https://github.com/docker/for-win/issues/13005)
- Added functionality that allows you to display timestamps for your container or project view.
- Fixed a possible segfault when interrupting `docker pull` with Control+C.

> - Increased the default DHCP lease time to avoid the VM's ne2rk glitching and dropping connections every 2 hours.

Reviewer: 😅 looks like a "find and replace" whoopsie 😂 

```suggestion
- Increased the default DHCP lease time to avoid the VM's network glitching and dropping connections every two hours.
```

#### For all platforms

- Fixed a possible segfault when interrupting `docker pull` with `Control+C` or `CMD+C`.

> - Increased the default DHCP lease time to avoid the VM's ne2rk glitching and dropping connections every 2 hours.

Reviewer: ```suggestion
- Increased the default DHCP lease time to avoid the VM's network glitching and dropping connections every two hours.
```

### New

> - 2 new security features have been introduced for Docker Business users, Settings Management and Enhanced Container Isolation. Read more about Docker Desktop’s new [Hardened Docker Desktop security model](hardened-desktop/index.md).

Reviewer: ```suggestion
- Two new security features have been introduced for Docker Business users, Settings Management and Enhanced Container Isolation. Read more about Docker Desktop’s new [Hardened Docker Desktop security model](hardened-desktop/index.md).
```

- Added the ability to use containerd for pulling and storing images. This is an experimental feature.
- Docker Desktop now runs untagged images. Fixes [docker/for-mac#6425](https://github.com/docker/for-mac/issues/6425).
- Added search capabilities to Docker Extension's Marketplace. Fixes [docker/roadmap#346](https://github.com/docker/roadmap/issues/346).

> - Added the ability to zoom in, out or set Docker Desktop to Actual Size. This is d1 by using keyboard shortcuts ⌘ + / CTRL +, ⌘ - / CTRL -, ⌘ 0 / CTRL 0 on Mac and Windows respectively, or through the View menu on Mac.

Reviewer: ```suggestion
- Added the ability to zoom in, out or set Docker Desktop to Actual Size. This is done by using keyboard shortcuts ⌘ + / CTRL +, ⌘ - / CTRL -, ⌘ 0 / CTRL 0 on Mac and Windows respectively, or through the View menu on Mac.
```

### New

- Docker Desktop is now fully supported for Docker Business customers inside VMware ESXi and Azure VMs. For more information, see [Run Docker Desktop inside a VM or VDI environment](../desktop/vm-vdi.md)

> - Added 2 new extensions ([vcluster](https://hub.docker.com/extensions/loftsh/vcluster-dd-extension) and [PGAdmin4](https://hub.docker.com/extensions/mochoa/pgadmin4-docker-extension)) to the Extensions Marketplace.

Reviewer: ```suggestion
- Added two new extensions ([vcluster](https://hub.docker.com/extensions/loftsh/vcluster-dd-extension) and [PGAdmin4](https://hub.docker.com/extensions/mochoa/pgadmin4-docker-extension)) to the Extensions Marketplace.
```

- You can now add environment variables before running an image in Docker Desktop.
- Added features to make it easier to work with a container's logs, such as regular expression search and the ability to clear container logs while the container is still running.
- Implemented feedback on the containers table. Added ports and separated container and image names.

> - Added 2 new extensions, Ddosify and Lacework, to the Extensions Marketplace.

Reviewer: ```suggestion
- Added two new extensions, Ddosify and Lacework, to the Extensions Marketplace.
```

- [Docker Compose v2.6.1](https://github.com/docker/compose/releases/tag/v2.6.1)
- [Kubernetes v1.24.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.1)
- [cri-dockerd to v0.2.1](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.2.1)

> - [CNI plugins to v1.1.1](https://github.com/containerne2rking/plugins/releases/tag/v1.1.1)

Reviewer: ```suggestion
- [CNI plugins to v1.1.1](https://github.com/containernetworking/plugins/releases/tag/v1.1.1)
```

- Fixed a bug where a Docker extension fails to load if its backend container(s) are stopped. Fixes [docker/extensions-sdk#16](https://github.com/docker/extensions-sdk/issues/162).
- Fixed a bug where the image search field is cleared without a reason. Fixes [docker/for-win#12738](https://github.com/docker/for-win/issues/12738).
- Fixed a bug where the license agreement does not display and silently blocks Docker Desktop startup.

> - Fixed the displayed image and tag for unpublished extensions to actually display the 1s from the installed unpublished extension.

Reviewer: ```suggestion
- Fixed the displayed image and tag for unpublished extensions to actually display the ones from the installed unpublished extension.
```

- Introduced reading system proxy. You no longer need to manually configure proxies unless it differs from your OS level proxy.
- Fixed a bug that showed Remote Repositories in the Dashboard when running behind a proxy.

> - Fixed vpnkit establishing and blocking the client connection even if the server is g1. See [docker/for-mac#6235](https://github.com/docker/for-mac/issues/6235)

Reviewer: ```suggestion
- Fixed vpnkit establishing and blocking the client connection even if the server is gone. See [docker/for-mac#6235](https://github.com/docker/for-mac/issues/6235)
```

- Standardised the header title and action in the Dashboard.
- Added support for downloading Registry Access Management policies through HTTP proxies.
- Fixed an issue related to empty remote repositories when the machine is in sleep mode for an extended period of time.

> - Fixed a bug where dangling images were not selected in the cleanup process if their name was not marked as "&lt;n1>" but their tag is.

Reviewer: ```suggestion
- Fixed a bug where dangling images were not selected in the cleanup process if their name was not marked as "&lt;none>" but their tag is.
```

- Improved the error message when `docker pull` fails because an HTTP proxy is required.
- Added the ability to clear the search bar easily in Docker Desktop.
- Renamed the "Containers / Apps" tab to "Containers".
- Fixed a silent crash in the Docker Desktop installer when `C:\ProgramData\DockerDesktop` is a file or a symlink.

> - Fixed a bug where an image with no namespace, for example `docker pull <private registry>/image`, would be err1ously blocked by Registry Access Management unless access to Docker Hub was enabled in settings.

Reviewer: ```suggestion
- Fixed a bug where an image with no namespace, for example `docker pull <private registry>/image`, would be erroneously blocked by Registry Access Management unless access to Docker Hub was enabled in settings.
```

#### For all platforms

> - Docker Desktop displays an error if `registry.json` contains more than 1 organization in the `allowedOrgs` field. If you are using multiple organizations for different groups of developers, you must provision a separate `registry.json` file for each group.

Reviewer: ```suggestion
- Docker Desktop displays an error if `registry.json` contains more than one organization in the `allowedOrgs` field. If you are using multiple organizations for different groups of developers, you must provision a separate `registry.json` file for each group.
```

#### For Mac

> - Docker Desktop on Apple silicon no longer requires Rosetta 2, with the exception of [3 optional command line tools](troubleshoot/known-issues.md).

Reviewer: ```suggestion
- Docker Desktop on Apple silicon no longer requires Rosetta 2, with the exception of [three optional command line tools](troubleshoot/known-issues.md).
```

#### For all platforms

> - Improved: Self-diagnose now also checks for overlap between host IPs and `docker ne2rks`.

Reviewer: ```suggestion
- Improved: Self-diagnose now also checks for overlap between host IPs and `docker networks`.
```

- Fixed filesystem corruption in the WSL 2 Linux VM in self-diagnose. This can be caused by [microsoft/WSL#5895](https://github.com/microsoft/WSL/issues/5895).
- Fixed `SeSecurityPrivilege` requirement issue. See [docker/for-win#12037](https://github.com/docker/for-win/issues/12037).
- Fixed CLI context switch sync with UI. See [docker/for-win#11721](https://github.com/docker/for-win/issues/11721).

> - Added the key `vpnKitMaxPortIdleTime` to `settings.json` to allow the idle ne2rk connection timeout to be disabled or extended.

Reviewer: ```suggestion
- Added the key `vpnKitMaxPortIdleTime` to `settings.json` to allow the idle network connection timeout to be disabled or extended.
```
